File: program337.cpp
/*
OUTPUT:
Enter the number:
1234
Number is:1234
Sum of digit is:10

Enter the number:
23
Number is:23
Sum of digit is:5

*/

#include<iostream>
using namespace std;

class Number
{
	private:
		int iNo;
		
	public:
	
		void Accept();
		void Display();
		int DigitSum();
};

void Number::Accept()
{
	cout<<"Enter the number:"<<endl;
	cin>>iNo;
}

void Number::Display()
{
	cout<<"Number is:"<<iNo<<endl;
	
}

int Number::DigitSum()
{
	int iDigit=0;
	int iSum=0;
	
	if(iNo<0)
	{
		iNo=-iNo;
	}
	
	while(iNo!=0)
	{
		iDigit=iNo%10;
		iSum=iSum+iDigit;
		iNo=iNo/10;
	}
	return iSum;
}

int main()
{
	
	Number nobj;
	int iRet=0;
	
	nobj.Accept();
	nobj.Display();
	iRet=nobj.DigitSum();
	cout<<"Sum of digit is:"<<iRet<<endl;
	
	return 0;
}

File: program337.java
import java.lang.*;
import java.util.*;
import MarvellousMatrix.Matrix;

class MyMatrix extends Matrix
{
	public MyMatrix(int a,int b)
	{
		super(a,b);
	}
	
	public int Maximum()
	{
		int iMax=Arr[0][0];
		
		int i=0,j=0;
		
		for(i=0;i<iRow;i++)
		{
			for(j=0;j<iCol;j++)
			{
				if(Arr[i][j]>iMax)
				{
					iMax=Arr[i][j];
				}
			}
			
		}
		
		return iMax;
	}
	
}

class program337
{
	public static void main(String arg[])
	{
		Scanner sobj=new Scanner(System.in);
		System.out.println("Enter the number of rows:");
		int X=sobj.nextInt();
		
		System.out.println("Enter the number of columns:");
		int Y=sobj.nextInt();
		
		MyMatrix mobj=new MyMatrix(X,Y);
		mobj.Accept();
		mobj.Display();
		int iRet=mobj.Maximum();
		System.out.println("Maximum number is:"+iRet);
	}
}
